index.html

```diff
@@ -49,16 +49,17 @@
 
         <button>Donate Now</button>
 
-    </div>
+    </header>
 
     <div class="content">
-        <div class="main">
+        <main class="main">
           <div class="alert">
             You've helped us give <span class="text-highlight">72</span> bikes
             to refugees and asylum seekers so far. <strong>Thank you!</strong>
           </div>
 
           <div class="hero">
+            
             <h1>Bikes for Refugees</h1>
```

Comment:
Please take a look at the buttons on the target image - can we make them look different? (one is white, another is brown/orange)